stanford/roles/cluster/defaults/main.yml

```diff
@@ -537,3 +543,46 @@ CLUSTER_SERVICE_TYPE: 'ec2'
 # alternate values are
 # CLUSTER_SERVICE_TYPE: 'rds'
 # CLUSTER_SERVICE_TYPE: 'emr'
+cluster_emr_internal_group_rules: &cluster_emr_internal_group_rules
```

Comment:
I don't think you can use the same variable name for both left-hand and right-hand sides here.

In the `edxapp` role, this is defined like:

```yaml
edxapp_generic_auth_config:  &edxapp_generic_auth
  TRACKING_BACKENDS: "{{ EDXAPP_TRACKING_BACKENDS }}"
  # etc...
```

and then referenced like:

```yaml
lms_auth_config:
  <<: *edxapp_generic_auth
  # etc...
```

When doing the interpolation, I think you need to use the variable name from the RHS.
I recall being bit on this in the past. So we can try something like:

```yaml
_cluster_emr_internal_group_rules: &cluster_emr_internal_group_rules
  - proto: icmp  # etc...
```

Notice, I just prefixed the LHS with an underscore `_`.
Or if you want, give the LHS a better name, though it shouldn't matter much, since we aren't referencing it elsewhere. As long as you keep RHS named the same, I think everything else should plumb though.